Core/include/Acts/Propagator/ConstrainedStep.hpp

```diff
@@ -131,6 +131,10 @@ struct ConstrainedStep {
 
   /// return the split value as string for debugging
   std::string toString() const;
+
+  /// Number of iterations needed by the stepsize finder
+  /// (e.g. Runge-Kutta) of the stepper.
+  int nStepTrials = -1;
```

Comment:
I see, logging this in the constrained step is actually a got idea.

Reply:
Probably safer to set to `std::numeric_limits<int>::max()`?